vendor/service-inject-3.js

```diff
@@ -1,13 +1,21 @@
 /* globals Ember */
 /* eslint-disable ember/new-module-imports */
 
+const { isElementDescriptor } = Ember.__loader.require('@ember/-internals/metal');
```

Comment:
I don't think we can rely on this module + method existing (this is likely why CI is failing for older Ember versions)

Reply:
I see! Added some logic in tests to only run if the version is greater than 3.10, and check to see if getElementDescriptor is present before invoking.